.helm/values.production.yaml

```diff
@@ -1,11 +1,15 @@
 appEnv: production
 djangoSettingsModule: reopt_api.production_settings
 djangoReplicas: 10
+djangoCpuRequest: "1000m"
+djangoCpuLimit: "2000m"
 djangoMemoryRequest: "2000Mi"
 djangoMemoryLimit: "2000Mi"
 celeryReplicas: 10
 celeryMemoryRequest: "900Mi"
```

Comment:
Need to bump this up to 1600Mi with the new python setup after seeing OOM issues on staging with the 700